dace/codegen/targets/intel_fpga.py

```diff
@@ -1497,6 +1497,20 @@ def visit_Assign(self, node):
 
         return ast.copy_location(updated, node)
 
+    def visit_BinOp(self, node):
+        if node.op.__class__.__name__ == 'Pow':
```

Comment:
Is pow the only math function we could encounter? Can't we look for any math function?

Reply:
No, is not the only one. But I was mimicking cppunparse, where Pow is a very special case and the `dace::math` string is printed out (cppunparse is called after OCL Keyword remover) https://github.com/spcl/dace/blob/b727e198e6fa8a8adf68b4c213365555ef1edef6/dace/codegen/cppunparse.py#L896